### DISCUSSION TITLE
Appreciation and Potential Requests for the MAISI Project

### DISCUSSION THREAD

**Discussion by gexinh** [Project-MONAI/tutorials#1866] [General]
As a PhD student at the University of British Columbia focusing on generative AI in medical imaging, I want to express my appreciation for the Maisi Project. My team previously worked on applying 3D Latent Diffusion Models (LDM) and ControlNet to medical datasets. At that time, we couldn’t find a framework that met our needs, so we had to modify 2D models ourselves, facing significant challenges in training Variational Autoencoders (VAE) and LDMs on medical data. MAISI has streamlined our workflow, making the entire process far more efficient. Its **well-designed architecture** and **tailored preprocessing and data augmentation** for medical datasets enable quick generalization across dataset with different modalities . The VAE and LDM components are **especially easy to use**, allowing us to train high-quality models with minimal tuning.

It would be great to see MAISI support other conditional LDMs, like the **T2I-Adapter**, which would enhance its utility in highly customized tasks like medical data. Maisi’s straightforward, ready-to-deploy framework is a huge asset. Thank you for creating such an impactful project!

**Comment by Can-Zhao** [comment]
I really appreciate your feedback and the suggestion! We are glad that it helped save your effort and accelerate the research. We will keep improving MAISI to better facilitate users' needs.
